integration-tests/billing/services/returns-loader.js

```diff
@@ -18,13 +19,21 @@ const createConnector = tail => async body => {
 };
 
 const create = () => {
+  // create a future date and pass it to the fixture loader as reference for use in yaml objects
+  const refDates = {
+    name: '$returnDates',
+    obj: {
+      dueDate: moment().add(1, 'month').format('YYYY-MM-DD')
```

Comment:
This is fine, as long as we don't care about the actual due date...(?)

Reply:
Thanks - yes I added an additional return specifically for this test scenario to ensure the status will always show as due on the ui. If the due date needs to be tested then the same logic could be used in Cypress or an alternative return with a static due date could be checked.